sys/arm64/arm64/copyinout.S

```diff
@@ -224,3 +224,128 @@ ending:
 	mov	x0, xzr		/* return 0 */
 	ret
 	.size	copycommon, . - copycommon
+
+#if __has_feature(capabilities)
+/*
+ * Copy specified amount of data from the user to kernel space, preserving
+ * capability tags
+ *
+ * int copyincap(const void * __capabilityuaddr, void *kdaddr, size_t len)
```

Comment:
```suggestion
 * int copyincap(const void *__capability uaddr, void *kdaddr, size_t len)
```

Reply:
Seems like there are 650 cases of `foo *const` and 2910 cases of `foo * const`. I guess clang-format needs an `SpaceBeforePointerQualifer: true` option :(

Reply:
although it seems a lot of that is contrib code, so yeah mostly consistent.

Reply:
Yeah, although it needs to only emit one space for PAS_Left and PAS_Middle.